packages/hint-babel-config/tests/is-valid.ts

```diff
@@ -62,7 +62,7 @@ const tests: HintLocalTest[] = [
         reports: [
             {
                 message: `Circular reference found in file ${path.join(__dirname, 'fixtures', 'circular-2', '.babelrc')}`,
-                position: { match: 'extends' }
+                position: { match: '"../circular-2/.babelrc"' }
```

Comment:
Note: Once @sarvaje adds range support to tests per https://github.com/webhintio/hint/pull/3138/files#r338139101 we should be able to use that here. Though we may want to make `match` optional for cases like this where the `range` and `match` are going to be identical.